src/plugins/intel_npu/src/plugin/npuw/serialization.cpp

```diff
@@ -74,6 +74,12 @@ void ov::npuw::s11n::write(std::ostream& stream, const ov::Output<const ov::Node
     write(stream, var.get_names());
 }
 
+void ov::npuw::s11n::write(std::ostream& stream, const ov::Any& var) {
+    // FIXME: figure out a proper way to serialize Any (for config)
+    NPUW_ASSERT(var.is<std::string>() && "Can't serialize ov::Any which is not std::string!");
```

Comment:
Please use `ov::Any::print(ostream)` function instead, as I am not sure that all properties we have are exactly strings

Reply:
Doesn't work for us due to it's usage of >> and << operators. It will read the whole file

Reply:
Tested several int types - works fine

Reply:
I don't understand, we can serialize in some `std::stringstream` and then to the ov::Any. Have you tested `ov::Any` with different types and they don't fail on `var.is<std::string>()`?

Reply:
> Doesn't work for us due to it's usage of >> and << operators. It will read the whole file

for read maybe it will read whole file, but for write should be ok?

Reply:
I need to align the behavior on both ends. I don't think it's a good idea to use `print()` which I don't own and my custom reader on the other side

Reply:
> I don't understand, we can serialize in some `std::stringstream` and then to the ov::Any. Have you tested `ov::Any` with different types and they don't fail on `var.is<std::string>()`?

I tried that approach. For some reason doesn't work. When I used `read(stringstream)` `ov::Any` object was empty - couldn't read from there.

Reply:
Discussed locally